ReplayKitExample/ReplayKitExample/ViewController.swift

```diff
@@ -55,6 +55,13 @@ class ViewController: UIViewController, RPBroadcastActivityViewControllerDelegat
         broadcastButton.layer.cornerRadius = 4
         conferenceButton?.layer.cornerRadius = 4
 
+        self.navigationController?.navigationBar.barTintColor = UIColor.init(red: 226.0/255.0,
+                                                                             green: 29.0/255.0,
+                                                                             blue: 37.0/255.0,
+                                                                             alpha: 1.0)
```

Comment:
It would be nice if we have had a utility which gives twilio red. (Out of scope for this PR though)